timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/LockWatchValueIntegrationTest.java

```diff
@@ -128,6 +128,7 @@ public void before() {
         LockWatchIntegrationTestUtilities.awaitTableWatched(txnManager, TABLE_REF);
     }
 
+    @ShouldRetry
```

Comment:
I think you can put this at the class-level? (It seems like this is on almost all if not all the methods.) You can override this to be 15 specifically on that one test

Reply:
It is already on this class which I've missed!

Overriding to 15 for all tests given that it retries for all the tests